data.json

```diff
@@ -1264,5 +1264,11 @@
     "githubUsername": "FayasNoushad",
     "imageUrl": "https://user-images.githubusercontent.com/76828314/129467637-18e9f157-4038-4aed-83e7-7d55e239c03f.jpg",
     "issueNumber": 646
+  },
+  {
+    "name": "Shubham Kukreti",
+    "githubUsername": "KukretiShubham",
+    "imageUrl": "https://user-images.githubusercontent.com/57281769/129481284-fe84223a-f3e4-41d8-9460-009254407e37.png",
```

Comment:
I suggest more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/129485691-4a1249b6-3203-4f15-ac8c-654a67d50ee4.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/129485691-4a1249b6-3203-4f15-ac8c-654a67d50ee4.png",
```

Reply:
Like this suggestion I would recommend going with something like this